apps/web/src/component/Auth/SignInForm.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Stack, TextField } from "@mui/material";
+import { FunctionComponent } from "react";
+import { Controller } from "react-hook-form";
+import { ThirdPartySignIn } from "./ThirdPartySignIn";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { NextAuthCredentials } from "@hublr/types";
+
+interface SignInFormProps {
+  isLoading: boolean;
+  onSubmit: (data: NextAuthCredentials) => Promise<void>;
+}
+
+const schema = yup
+  .object({
+    email: yup.string().email().required("Required"),
+    password: yup
+      .string()
+      .required("Required")
+      .matches(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/,
+        "Must Contain at least 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character of !@#$%^&*"
+      ),
```

Comment:
```suggestion
      .required("Required"),
```

we don't need the match if user is signing in